src/selectors/entities/roles_helpers.ts

```diff
@@ -14,6 +14,10 @@ export type PermissionsOptions = {
     permission: string;
 };
 
+export type SysConsoleItemOptions = {
+    resourceId: string;
+};
+
```

Comment:
As far as i can tell this isnt used in redux at all does it make more sense to just keep it on the webapp side?

Reply:
Good point, and it's not used in webapp either. I've removed it. Thanks.